### PR TITLE
Mnrva 601

### PR DESCRIPTION
**Pull request by prit4321**
## JIRA: 
See ticket here - https://jira.rax.io/browse/MNRVA-601

 ## Description: Added the allure report

 ## Testing: npm run test:e2e

 ## Screenshots: 
(if applicable)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,10 +47,12 @@
     "@ngx-pwa/local-storage": "9.0.3",
     "@webcomponents/webcomponentsjs": "2.1.3",
     "ajv": "6.12.0",
+    "allure-commandline": "^2.13.0",
```
Comment (pras7430) [inline package.json:50]: the package should have a fixed version only

File: package.json [hunk 2]
```diff
@@ -46,10 +47,12 @@
...
     "chromedriver": "2.45.0",
     "core-js": "2.5.4",
     "hedwig-monitoring-library": "https://github.com/racker/Hedwig/archive/v0.1.0.tar.gz",
     "helix-ui": "1.1.0-rc.0",
+    "jasmine-allure-reporter": "^1.0.2",
```
Comment (pras7430) [inline package.json:55]: here too!!

**Review by pras7430 — COMMENTED** [review]
Allure packages should be in dev dependencies?

File: package.json (hunk 1 above)
Comment (prit4321) [reply package.json:50]: Yes, allure packages should be in dev dependencies. I will do the changes.

Comment (prit4321) [reply package.json:50]: Both the allure command line and jasmine allure reporter version are the latest one. Fixed version?

Comment (prit4321) [reply package.json:50]: I am also going to change the Buffer version as it's a deprecated one.

Comment (pras7430) [reply package.json:50]: it should be like below

"allure-commandline": "2.13.0"

Comment (prit4321) [reply package.json:50]: Okay

**Review by prit4321 — COMMENTED** [review]
Please review

**Pull request merged**